### DISCUSSION TITLE
Preferred PHP version

### DISCUSSION THREAD

**Discussion by kour1er** [Z-Hub/Z-Push#121] [Q&A] [answered]
I'm working on creating a Docker container (Alpine) for Z-Push and was wondering what is the preferred version of PHP for best results with Z-Push? 

Thanks for any pointers

**Comment by matidau** [comment]
While PHP 7.4 is the minimum version supported, I would recommend using a PHP 8.x version that has the longest support cycle for your distro. I think for Alpine this is PHP 8.1, but you are best checking that.

**Reply by kour1er** [reply, marked as answer]
Great - thanks for the pointer - 8.1 it is :)

**Comment by matidau** [comment]
And would be keen to hear more on your docker container when you're ready to share 😊

**Reply by kour1er** [reply]
Indeed - it's proving to be a little challenging at the moment - odd issue where Z-Push is pushing correctly (this is working on the [docker-mailserver](https://github.com/docker-mailserver/docker-mailserver) backend) but after 15 minutes or so the push connect seems to drop. So trying to figure out what's going on 🤔

**Reply by etique57** [reply]
@kour1er maybe you know but the original author of DMS did a container a few years ago, maybe it has some hints?

https://github.com/tomav/docker-z-push/tree/master

**Reply by kour1er** [reply]
thanks for the pointer - most kind!
